specs/credentialmanagement/index.src.html

```diff
@@ -238,7 +238,7 @@ <h4 id="examples-password-signin">Password-based Sign-in</h4>
     <div class="example">
       <pre>
         navigator.<a attribute>credentials</a>.<a idl lt="get()">get</a>({
-          "<a idl>types</a>": [ "password" ]
+          "<a idl>filters</a>": { "password" : {} }
```

Comment:
I don't like this syntax for `password`. Passing in an empty object isn't at all intuitive, and since I see this as the core use case the current API is attempting to enable, I'd like to ensure that it is as streamlined as possible.